core/src/test/java/nl/nn/adapterframework/parameters/ParameterTest.java

```diff
@@ -114,13 +114,18 @@ public void testPatternUnknownSessionVariableOrParameter() throws ConfigurationE
 		p.setName("dummy");
 		p.setPattern("{unknown}");
 		p.configure();
-		
+
 		PipeLineSession session = new PipeLineSession();
-		
+
 		ParameterValueList alreadyResolvedParameters=new ParameterValueList();
+
+		try {
+			p.getValue(alreadyResolvedParameters, null, session, false);
+			Assert.fail("supposed to throw an exception");
+		} catch(ParameterException e) {
+			assertEquals("Parameter or session variable with name [unknown] in pattern [{unknown}] cannot be resolved", e.getMessage());
+		}
```

Comment:
Was it necessary to change the code this way? I would say the previous implementation is more readable